README.md

```diff
@@ -37,16 +41,48 @@ const bmp = require('fast-bmp');
 const imageData = {
   width: 5,
   height: 5,
-  data: new Uint8Array([0b00000011, 0b10010100, 0b11100000, 0b00000000]),
-  bitDepth: 1,
+  data: new Uint8Array([
+    0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 1, 0, 1, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0,
```

Comment:
Are you able to format it so that 1 line represents one row in the image?

Reply:
Just to make sure. For the formatting of arrays, you want it only for the part in README or for the whole project? Because if it's for the whole project then I am not sure which is the way to do it. I can add comment to each row like `//Row X` , so that it keeps the row formatting. Or I can add `prettier-ignore` and format them the way i want to. Or I can pass data as 2D array to my test function and flatten it when creating an object. Or maybe there is a way to configure it in prettier, so that it formats the arrays correctly. I am leaning towards passing image data as 2D array and flatten it when creating an object.

Reply:
For tests `2Darray.flat()` will be used